CHANGELOG.md

```diff
@@ -1 +1,2 @@
+Add imperial support for UDCF import
```

Comment:
sadly, we didn't agree on `changelog.md` entry formatting in the mailing list discussion.
also, adding above `# add new entries above this line` and under your line `Add imperial support for UDCF import` (once it's already in) will probably break.

Reply:
So should I just let it be as is or change something?

Reply:
we prefix the entries with `-`:
`- Add imperial support for UDCF import`